indexer/entity/src/stake_delegation.rs

```diff
@@ -0,0 +1,38 @@
+use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel, Deserialize, Serialize)]
+#[sea_orm(table_name = "StakeDelegationCredentialRelation")]
+pub struct Model {
+    #[sea_orm(primary_key, column_type = "BigInteger")]
+    pub id: i64,
+    pub stake_credential: i64,
```

Comment:
shouldn't we add `#[sea_orm(column_type = "BigInteger")]`?

Reply:
I'm not sure that does anything, to be honest. The tables already have `bigint` type for me, and it's what the docs say should happen I think (https://www.sea-ql.org/SeaORM/docs/next/generate-entity/entity-structure/).

But I can add it just in case anyway.